package.json

```diff
@@ -376,13 +375,14 @@
         "@types/semver": "^5.5.0",
         "@types/tcp-port-used": "^1.0.0",
         "@types/uuid": "^3.4.4",
+        "@types/vscode": "^1.31.0",
```

Comment:
Should we take this opportunity to bump to a higher version?

Reply:
No, that is beyond the scope of responsibilities in this change